manifests/init.pp

```diff
@@ -93,8 +93,9 @@
     validate_absolute_path($ssl_cert_db, $ssl_cert_password_file)
   }
 
-  class { '::qpid::install': } ~>
-  class { '::qpid::config': } ~>
-  class { '::qpid::service': } ->
-  Class['qpid']
+  include ::qpid::install
```

Comment:
That results into a dependency cycle. And since we contain the service which in turn requires the package that should all work out.